index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+<meta charset="utf-8">
+<link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+
+<h1>todos</h1>
+
+<div class="table">
+	<div class="div_first"><label> ˅ </label><em> What needs to be done?</em></div>
```

Comment:
please update class names, no need to use word 'div'

Reply:
fixed